src/utils.js

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Utils is a pretty vague name for this, though I'm not sure I have a better one onhand. `tagproUtils` is not much more descriptive necessarily, although these utilities are all for directly interfacing with `tagpro`. `tagproInterface`? `interface`? `gameInterface`?